CMakeLists.txt

```diff
@@ -21,6 +21,10 @@ endif()
 ######## Build configuration options ########
 set(BUILD_EXAMPLE_APPLICATIONS ON CACHE BOOL "Build example applications.")
 set(BUILD_MBEDTLS ON CACHE BOOL "Build Mbed TLS library. When set to OFF, the Mbed TLS library with the OCF patches has to be provided.")
+set(OC_INSTALL_MBEDTLS ON CACHE BOOL "Include Mbed TLS in installation")
```

Comment:
when I use this flags ?

Reply:
It changes the behavior of `make install` (or `cmake --install`). When `OC_INSTALL_TINYCBOR` and `OC_INSTALL_MBEDTLS` is OFF then only iotivity-lite targets are installed. When they are on then also tinycbor and mbedtls targets are installed.